### PR TITLE
Update _fluent.scss

### PR DESCRIPTION
**Pull request by ig-georgeA**
Recommending some changes to the fluent type scale based on what we have in our Indigo.Design UI kit for Fluent, which closer to what MSFT has. Andrea did not have access to make a branch.

### CODE REVIEW

**Comment by simeonoff** [comment]
I believe @sdimchevski must approve this first. Before we can merge and release any changes introduces in this PR, a new version of the UI kits with the changes made here must be released, or at least released at the same time.

**Comment by sdimchevski** [comment]
@ig-georgeA great suggestion for these changes. We are working on Indigo.Design UI kit for Fluent in Figma now and we'll update these font-weights and letter-spacings. @simeonoff we should sync on the release as we have a way to go with the Figma kit and no resources for Sketch at the moment

**Review by sdimchevski — DISMISSED** [review]

File: sass/typography/presets/_fluent.scss [hunk 1]
```diff
@@ -4,13 +4,17 @@
 /// @group Typography
 ////
 
+/* Typescale reference: https://developer.microsoft.com/en-us/fluentui#/styles/web/typography
+Also used in our Figma UI Kit for Fluent: https://www.figma.com/file/hHXKBpg1UhCMziIkdsYLFF/Indigo.Design-for-Fluent---Workfile?node-id=1602%3A134705&t=4JiRXrp6ifvJVMtO-1
```
Comment (ig-georgeA) [inline sass/typography/presets/_fluent.scss:8]: Is it this line that's preventing causing the build to fail? @simeonoff

Comment (ig-georgeA) [reply sass/typography/presets/_fluent.scss]: I removed the line and made another commit.

**Review by sdimchevski — APPROVED** [review]

**Comment by simeonoff** [comment]
@sdimchevski What's the status of the changes introduced in this PR with regards to the UI kits? Is it safe to merge it?

**Comment by simeonoff** [comment]
Closing as the typography was revised by Svilen's Team and is inline with what we have in the UI kits.

**Pull request closed without merging**